applications/lfrfid/helpers/decoder_ioprox.cpp

```diff
@@ -0,0 +1,95 @@
+#include "decoder_ioprox.h"
+#include <furi_hal.h>
+#include <cli/cli.h>
+#include <utility>
+
+constexpr uint32_t clocks_in_us = 64;
+
+constexpr uint32_t jitter_time_us = 20;
+constexpr uint32_t min_time_us = 64;
+constexpr uint32_t max_time_us = 80;
+
+constexpr uint32_t min_time = (min_time_us - jitter_time_us) * clocks_in_us;
+constexpr uint32_t mid_time = ((max_time_us - min_time_us) / 2 + min_time_us) * clocks_in_us;
+constexpr uint32_t max_time = (max_time_us + jitter_time_us) * clocks_in_us;
+
+constexpr uint32_t baud_time = 32000;
```

Comment:
Please express the baud rate in something meaningful, magic numbers are not welcome.

Reply:
Fixed in the most recent patch-set.